combat-trainer.lic

```diff
@@ -2643,7 +2646,7 @@ class AttackProcess
         use_charged_maneuver(game_state.selected_maneuver, game_state)
         game_state.action_taken
       else
-        command = if game_state.use_stealth_attack? && hide?(@hide_type)
```

Comment:
Yes i was thinking to separate it in to 2 pieces

Reply:
Will this change cause any problems for those currently using this setting?

Reply:
It would require people who want to use stealth attacks with ranged weapons to change the setting.

Reply:
@rpherbig Is this a big problem?

Reply:
I'm trying to think of a way to avoid disrupting existing users.

Maybe we maintain the existing functionality if `use_stealth_attacks: true`, and then you add `use_stealth_ranged` and `use_stealth_melee` to be more specific? With a validator to deprecate `use_stealth_attacks`.

Is that too convoluted?

Reply:
how about something like this? 
command = if game_state.use_stealth_ranged? && hide?(@hide_type)  
                    @stealth_attack_aimed_action
                  elsif game_state.use_stealth_attack? && hide?(@hide_type)
				    @stealth_attack_aimed_action
			      else
                    'shoot'
                  end
        shoot_aimed(command, game_state)
      end
      game_state.loaded = false
      waitrt?

Reply:
I tried this command structure and it seems to be working well.  Aruthas has stealth_attacks: false
and my range has them true.  Should I add this change to the commit?